packages/blocks/src/api/raw-handling/anchor-reducer.js

```diff
@@ -0,0 +1,25 @@
+export default function anchorReducer( node ) {
```

Comment:
This is just separated out from https://github.com/WordPress/gutenberg/pull/62659/files#diff-56e921b5ac7d8092c4085c3311a2611048fbd44a11f49de8e307bd796a4cb671